stock-tracker/src/App.js

```diff
@@ -2,25 +2,24 @@ import React from "react";
 import Header from "./components/Header";
 import KeyStats from "./components/keyStats";
 import Chart from "./components/chart";
-import LatestNews from './components/LatestNews';
-import Overview from './components/Overview';
-import TopPeers from './components/TopPeers';
-import Footer from './components/Footer';
-import './App.css';
-import { useSelector } from 'react-redux'
+import LatestNews from "./components/LatestNews";
+import Overview from "./components/Overview";
+import TopPeers from "./components/TopPeers";
+import Footer from "./components/Footer";
+import "./assets/styles/App.css";
```

Comment:
This import has changed, which highlights to me that this PR has more than just `prettier` ran on it...

Reply:
A bunch of new `.css` files also exist.